packages/styles/scss/components/leadspace/_tokens.scss

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright IBM Corp. 2020
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '@carbon/colors' as *;
+@use '@carbon/styles/scss/themes';
+@use '@carbon/styles/scss/theme' as *;
+@use '@carbon/styles/scss/utilities/component-tokens';
+
+$btn-min-width: 26;
+
+$leadspace-heading-highlight: (
```

Comment:
I think all of the highlight color token logic could be removed by simply using `$link-primary`, no? See the [Link](https://carbondesignsystem.com/guidelines/color/tokens) section of the color guidance page.

Reply:
That's true! I didn't realize there was already a token for that color mapping. I'll remove the token file and use this token instead

fyi @RichKummer, the [specs](https://www.figma.com/file/Gfb6xdHwKdBgNloNqlSbnG/Lead-space-v2?type=design&node-id=1021-36034&mode=design&t=lGZeOZtq02WhoAAA-4) can be updated to include the token instead 😄 👍